Marlin/src/HAL/HAL_STM32F1/HAL_timers_STM32F1.cpp

```diff
@@ -57,7 +57,7 @@ void HAL_timer_start(const uint8_t timer_num, const uint32_t frequency) {
     case 3: irq_num = NVIC_TIMER3; break;
     case 4: irq_num = NVIC_TIMER4; break;
     case 5: irq_num = NVIC_TIMER5; break;
-    #ifdef STM32_HIGH_DENSITY
+    #if ENABLED(STM32_HIGH_DENSITY)
```

Comment:
may "need"  || defined(STM32_XL_DENSITY) here... but timer 8 is not used in marlin

Reply:
*not yet(?)

Reply:
could be used as extra servos.. (2 per adv timer)